### ISSUE TITLE
Relative imports are relative to the path where the code was called not relative to the file position

### ISSUE THREAD

**Issue by twolodzko** [uiua-lang/uiua#220]
Consider the example, where there are two files with the structure:

first.ua:
```
A ← 1
```
inner/second.us
```
AA ← &i "../first.ua" "A"
---
⍤ 1 AA
---
```

If you call the code from the root directory you get:
```
$ uiua test inner/second.ua
Error: No such file or directory (os error 2)
  at inner/second.ua:1:6
1 | AA ← &i "../first.ua" "A"
```

and from the inner directory it works

```
$ cd inner
$ uiua test second.ua      
No failures!
```

The same goes for `uiua watch`. 

This may be intentional or a bug, but I had a hard time finding any documentation mentioning this. It is also a bit confusing, I'd assume that the relative path is the relative to where the file that does the import is placed not where it is called from. When defined as it is right now, it can lead to some cryptic bugs where the same code imports different things depending on how it was called.

**Comment by kaikalii** [comment]
I'm aware of this.
I'm not sure if I consider it a bug.

**Comment by twolodzko** [comment]
It could be problematic e.g. the tests in CI pass, but when you call the same code from a different directory where the CI script was located, it does not work anymore.

**Comment by kaikalii** [comment]
That's a good point.
I've fixed this in e56400012c712ef.

**Comment by teeth-eator** [comment]
C and python also list their includes/imports relative to the source file so I think this is a good change
